_includes/html/desktop-table.html

```diff
@@ -98,7 +106,14 @@
                   <i class="tfa-icon fas fa-check col"></i>
                 {% endif %}
                 {% if tfa contains "custom-software" %}
-                  <i class="tfa-icon fas fa-info col" title="Proprietary software based 2FA"></i>
+                  <i class="tfa-icon fas fa-info col"
+                     title="{%- if website.custom-software -%}
+                     Required app: {{ website.custom-software }}
```

Comment:
```suggestion
                     Required app:&#013;{%- for item in website.custom-software -%} {{ item }}&#013; {%- endfor -%}
```
Entries with multiple custom-* values look quite messy at the moment: 
![image](https://user-images.githubusercontent.com/20560218/133927330-f377720f-a1f4-4af7-8001-184af40ead61.png)

It may be worth adding new lines in between to make it a bit clearer:
![image](https://user-images.githubusercontent.com/20560218/133927346-83ebf48c-a85f-4e90-b47f-610b099fdd37.png)

Reply:
This seems to be better, since it doesn't use the final line break:
`Required app:{%- for item in website.custom-software -%}&#013;{{ item }}{%- endfor -%}`